python_modules/dagster/dagster/core/host_representation/repository_location.py

```diff
@@ -422,6 +430,13 @@ def get_external_partition_set_execution_param_data(
             partition_names=partition_names,
         )
 
+    def get_external_notebook_data(
+        self, repository_handle: RepositoryHandle, notebook_path: str
```

Comment:
I don't think you need repository_handle, see comment on the other diff. Thanks for checking!